packages/neovim/src/utils/findNvim.ts

```diff
@@ -170,14 +189,22 @@ function getPlatformPaths() {
  * @param opt.minVersion See {@link FindNvimOptions.minVersion}
  * @param opt.orderBy See {@link FindNvimOptions.orderBy}
  * @param opt.firstMatch See {@link FindNvimOptions.firstMatch}
+ * @param opt.paths See {@link FindNvimOptions.paths}
+ * @param opt.dirs See {@link FindNvimOptions.dirs}
  */
 export function findNvim(opt: FindNvimOptions = {}): Readonly<FindNvimResult> {
-  const paths = getPlatformPaths();
+  const platformDirs = getPlatformSearchDirs();
+  const nvimExecutable = windows ? 'nvim.exe' : 'nvim';
+
+  const allPaths = new Set<string>([
+    ...(opt.paths || []).map(normalizePath),
+    ...(opt.dirs || []).map(dir => normalizePath(join(dir, nvimExecutable))),
```

Comment:
|| is different than ??
```suggestion
    ...(opt.paths ?? []).map(normalizePath),
    ...(opt.dirs ?? []).map(dir => normalizePath(join(dir, nvimExecutable))),
```